plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchProxy.java

```diff
@@ -58,7 +58,7 @@
             "    proxy_set_header Proxy-Connection \"Keep-Alive\";\n" +
             "    client_max_body_size 0;\n" +
             "  }\n" +
-            "  location ~ /.*/_search {\n" +
+            "  location ~ /.*/_search$ {\n" +
```

Comment:
previous commit

Reply:
I wanted to do a squash before merge. The only reason I split this in two commits is that it makes multiple review rounds easier, since you know you've reviewed a commit already.